### PR TITLE
Simplecov

### PR DESCRIPTION
**Pull request by mtrojanowska**
:)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+coverage/*
+!coverage/index.html
```
Comment (ktopolski) [inline .gitignore]: Why add exception for the `coverage/index.html`? It's an unusual thing to do, I'm curious on the reasoning behind it :)

**Review by ktopolski — COMMENTED** [review]
This PR will be mostly ready to merge when the rspec one gets merged. Curious on the `coverage/index.html`, looking forward to the discussion!

File: .gitignore (hunk 1 above)
Comment (ktopolski) [reply .gitignore]: @mtrojanowska let's remove it as we agreed to do so on our phone conversation

**Review by ktopolski — APPROVED** [review]
Together with #1 we made these changes while sitting next to each other in cafe. Looks great now, ready to be merged :)

**Comment by ktopolski** [comment]
@mtrojanowska please feel free to merge approved PRs whenever you feel like it :) Please keep in mind to merge them in correct order, this one should go first.

**Comment by mtrojanowska** [comment]
First branch merged by me

**Pull request closed without merging**